woocommerce/woocommerce
# Migrating pricing section in product editor to slot fills

### All Submissions:






### Changes proposed in this Pull Request:



Transitioning pricing section in product editor to utilize our slot-fill components. 

Closes #36419  .




### How to test the changes in this Pull Request:



1. Checkout branch.
2. Enable product editor at WooCommerce -> Settings -> Advanced -> Features -> Try the new product editor
3. Go to product editor at Products -> Add new (MVP)
4. Do an overall smoke test of the pricing fields. Be sure you enable taxes and test the fields that only appear then.



### Other information:




### FOR PR REVIEWER ONLY:
